tests/unittests/CoreGraphics.drawing/CGContextBlendModeTests.cpp

```diff
@@ -131,7 +131,8 @@ static void __drawRightStackPink(CGContextRef context, CGRect bounds) {
     CGContextRestoreGState(context);
 }
 
-class CGContextBlendMode : public WhiteBackgroundTest<>, public ::testing::WithParamInterface<::testing::tuple<bool, CGNamedBlendMode>> {
+class CGContextBlendMode : public WhiteBackgroundTest<PixelByPixelImageComparator<PixelComparisonModeDifferenceLimen<100>>>,
```

Comment:
Mmhmm, there were between 1-75 pixels where the differences were beyond our threshold.